README.md

```diff
@@ -323,6 +323,41 @@ The documentation is rendered using an interactive terminal session that you can
 
 run `fastify docs` to get started.
 
+## Test helpers
+
```

Comment:
make it clear that the test helper are only for tap and not jest/mocha/tape etc.